### ISSUE TITLE
Lures are bugged

### ISSUE THREAD

**Issue by 23ttfe3fa** [AHAAAAAAA/PokemonGo-Map#2289]
## Steps to Reproduce (for bugs)
1. Find a lure
2. Refresh page
3. Lure has longer duration then before the reset
4. Refresh even more times over time causes timers to act weird

## Your Environment

* Version used:  Current at time of posting
* Environment name and version (e.g. Python 2.7): Python 2.7
* Operating System and version (desktop or mobile): Desktop

**Comment by Scyntrus** [comment]
Caused by this:
https://github.com/AHAAAAAAA/PokemonGo-Map/blob/develop/static/js/map.js#L491

**Comment by 23ttfe3fa** [comment]
Agreed, But what is the solution. In previous versions it was not ideal because lures refreshed on a 2minute loop I'm assuming because according to the server information lures weren't on a 30 minute timer but the servers where using fifteen 2 minute timers and we could only find the information for the current 2 minutes.

This change on the current version was meant to fix that because it was thought last_modified_date was last time it was updated from no lure to lure or lure to no lure, But it seems to be any modification such as refreshing.

Apparently the user of the lure can see a duration left, Maybe there is a solution there, I can't think of a solution to have accurate timers.
